src/Symfony/Component/Form/ButtonBuilder.php

```diff
@@ -79,9 +79,6 @@ public function __construct(?string $name, array $options = [])
      *
      * This method should not be invoked.
      *
-     * @param string|FormBuilderInterface $child
```

Comment:
@Tobion , why have you removed the `$child` phpdoc? It still gives useful information about the argument which cannot be replaced with a declaration.

Reply:
oops, that's the ButtonBuilder, then it's okay, sorry)